composite/proof-of-reserves/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "outDir": "dist",
+    "rootDir": "src",
+    "typeRoots": ["../../node_modules/@types", "../../typings", "./typings"]
+  },
+  "include": ["src/**/*"],
+  "exclude": ["dist", "**/*.spec.ts", "**/*.test.ts"]
```

Comment:
We should probably include the tests so they get type-checked, or create another tsconfig.json just for tests

Reply:
Ok, will keep this in mind for future TS improvements. I do not have the resources to change/test this now.